demos/twa-location-delegation/README.md

```diff
@@ -0,0 +1,7 @@
+# Trusted Web Activity / Location Delegation Demo
+
+This demo application shows how to make the TWA app uses Android location permission instead of
+browser permission:
+1. Adding the location delegation dependency on build.gradle
```

Comment:
Could you put full stops at the end of these points please?

Reply:
Done.